app/brave_main_delegate_browsertest.cc

```diff
@@ -27,7 +27,6 @@
 #include "components/reading_list/features/reading_list_switches.h"
 #include "components/security_state/core/features.h"
 #include "components/shared_highlighting/core/common/shared_highlighting_features.h"
-#include "content/public/browser/render_view_host.h"
```

Comment:
web_preferences.h ?

Reply:
Thanks! Indeed, that include is no longer needed. Removed.